db/migrate/20210803194547_add_show_guidance_speedbump_to_early_career_teacher_profile.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddShowGuidanceSpeedbumpToEarlyCareerTeacherProfile < ActiveRecord::Migration[6.1]
+  def change
+    add_column :early_career_teacher_profiles, :show_guidance_speedbump, :boolean, default: true
```

Comment:
Should we name the flag something less intent-ful and more state-ful?

`guidance_seen`? perhaps? 

That leads to another interesting point. If the user sees the guidance some other way than speedbump, should we still show them a speedbump?

Reply:
I went with something similar originally and then swapped to the above, as `guidance_seen` could be true before they've hit the speedbump. 

I've gone through a bunch of different names, all not that great

How about... `given_view_guidance_option? `

Reply:
`guidance_speedbump_seen`?

Reply:
Depending whether we decide to still show the speedbump or not i'll go with either `guidance_seen` or `guidance_speedbump_seen`

Reply:
We're going with `guidance_seen`